.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,10 @@
+---
+name: Bug report
+about: To report a part of mocha not working as expected
+title: ''
+labels: 'unconfirmed-bug'
+---
+
 <!--
 Have you read Mocha's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md
 For more, check out the Mocha Gitter chat room: https://gitter.im/mochajs/mocha
```

Comment:
Would be nice to mention providing an MCVE somewhere. I tire of having to ask afterwards.

Reply:
Could you provide a sentence or a template for MCVE because I'm not native for English.

Reply:
Detail the steps necessary to reproduce the problem. To get the fastest support, create an [MCVE](https://stackoverflow.com/help/mcve) and upload it to GitHub.